src/main/java/nextstep/subway/entity/GithubClient.java

```diff
@@ -0,0 +1,71 @@
+package nextstep.subway.entity;
+
+import nextstep.subway.dto.GithubAccessTokenRequest;
+import nextstep.subway.dto.GithubAccessTokenResponse;
+import nextstep.subway.dto.GithubProfileResponse;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.*;
+import org.springframework.stereotype.Component;
+import org.springframework.util.MultiValueMap;
+import org.springframework.web.client.RestTemplate;
+
+import java.util.Collections;
+
+@Component
+public class GithubClient {
+    @Value("${github.client.id}")
+    private String client;
+
+    @Value("${github.client.secret}")
+    private String clientSecret;
+
+    @Value("${github.url.token}")
+    private String githubUrlToken;
+
+    @Value("${github.url.user.profile}")
+    private String githubUrlUserProfile;
+
+    public String requestGithubToken(String code) {
+        GithubAccessTokenRequest githubAccessTokenRequest = new GithubAccessTokenRequest(code, client, clientSecret);
+
+        HttpEntity<MultiValueMap<String, String>> httpEntity = new HttpEntity(githubAccessTokenRequest, makeJsonHeader());
```

Comment:
```suggestion
        HttpEntity<GithubAccessTokenRequest> httpEntity = new HttpEntity<>(githubAccessTokenRequest, makeJsonHeader());
```

MultiValueMap 으로 하신 이유가 있을까요? 🤔